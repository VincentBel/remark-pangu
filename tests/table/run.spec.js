runSpec(__dirname);
runSpec(__dirname, {
  type: {
    text: false,
  },
});
