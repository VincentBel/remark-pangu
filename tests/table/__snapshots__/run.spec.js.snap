// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table.md: table.md 1`] = `
| header abc123 | header 我a我1 |
| :-- | :-: |
| cell abc123 | cell 我a我1 |
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
| header abc123 | header 我 a 我 1 |
| :------------ | :------------: |
| cell abc123   |  cell 我 a 我 1  |

`;
